src/schema.gql

```diff
@@ -114,6 +121,27 @@ type Following {
   resourceName: String!
 }
 
+type Institute {
+  _id: ID!
+  createdBy: User!
+  followerCounter: Float!
```

Comment:
@frankmangone shouldn't this be Number?  why the exclamation?

Reply:
You're right in that it should be an Int, so I'm fixing that for all the schemas. As for the exclamation mark, it means that it's a non-nullable field; this is expected behavior, since the cache should always have a non-null value